primitives/src/transaction.rs

```diff
@@ -529,6 +529,13 @@ impl Version {
 
     /// The third Bitcoin transaction version (post-BIP-431).
     pub const THREE: Self = Self(3);
+
+    /// Constructs a new [`Version`] from an u32
+    #[inline]
```

Comment:
This made me wonder why one function was inlined and not the other, then I saw that its copied from `block::Version`. Just as a matter of principle, if you see something you don't understand just mention it in the PR or commit log. In this case I see no reason and `block::Version` is wrong too IMO.

Reply:
Of course there is a chance that I just don't understand it either. But I think both should be inlined.

Reply:
Basically, if a function jut delegates into another function and passing through arguments, it should be inline. Also when doing assignment of `u32` or similarly trivial operation. Also if it checks the validity of argument(s) and then performs something simple (like a call to `_unchecked`).

Reply:
Nice! That is a way easier metric than 'less than some amount of lines of assembler' - WIN!